pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -109,6 +109,9 @@ func (a *Assignment) TotalRequestsFor(wl *workload.Info) cache.FlavorResourceQua
 	usage := make(cache.FlavorResourceQuantities)
 	for i, ps := range wl.TotalRequests {
 		for res, q := range ps.Requests {
+			if a.PodSets[i].Flavors == nil {
```

Comment:
We shouldn't be calling this function if we couldn't find an assignment.

The fix should be in `scheduler.go`. If `RepresentativeMode` is NotFit, we shouldn't calculate a resource share.

Reply:
Done